### ISSUE TITLE
Focus day on calendar appears to change when navigating w/ keyboard 

### ISSUE THREAD

**Issue by smcmurtry** [cds-snc/notification-scheduler#3]
Steps to reproduce:
- tab to calendar and use the arrow keys to move to the 20th day of the month
- select they day with the spacebar
- hit tab, to focus on the time dropdown, then shift-tab to go back to the calendar

At this point you should be focused on the 20th, since that is the last focused day on the calendar and probably closer to the next selection the user will make than the current day is). Instead you are focused on the current day of the month.

**Comment by timarney** [comment]
Fixed a few items here.

Next Month, Previous Month - should now give focus to the first available date in the corresponding month.

For example if you're in Feb and today is Jan 27th navigation to the Previous Month focuses on Jan 27th moving to Next Month takes you to Feb 1.

This should able correct the tab shift tab issue.  Navigate to the dropdown and back to the Calendar.

**Comment by timarney** [comment]
Closing this as I believe it's fixed.
